homepage/neuroimaging.rst

```diff
@@ -26,21 +26,32 @@ The technology and physics behind an MRI scanner is quite astonishing. But I won
+
+A standard anatomical volume, with an isotropic voxel resolution of 1mm contains almost 17 million voxels, which are arranged in a **3D matrix** of 256 x 256 x 256 voxels. The following picture shows a vertical slice -- one layer of the big, 3D grid looked at from the top -- through a brain volume, and the grid for the remaining two dimensions is superimposed on the picture of the brain.  The data is measured per box, and each of the boxes corresponds to a particular portion of the brain.
 
 .. image:: images/voxel.png
    :width: 300pt
    :align: center
 
-As the scanner can't measure the whole volume at once it has to measure it sequentially. This is done by measuring one plane of the brain  (generally the horizontal one) after the other. Such a plane is also called a **slice**. The resolution of the measured volume data therefore depends on the in-plane resolution, the number of slices, their thickness, and a possible gap between them.
+As the scanner can't measure the whole volume at once it has to measure portions of the brain sequentially in time. This is done by measuring one plane of the brain (generally the horizontal one) after the other. Such a plane is also called a **slice**. The **resolution** of the measured volume data, therefore, depends on the in-plane resolution (the size of the squares in the above image), the number of slices and their thickness (how many layers), and any possible gaps between the layers.
 
 The quality of the measured data depends on the resolution and the following parameters:
 
 * **repetition time (TR)**: time required to scan one volume
-* **acquisition time (TA)**: time required to scan one volume. TA = TR - (TR/number of slices)
+* **acquisition time (TA)**: time required to scan one slice. TA = TR - (TR/number of slices)
 * **field of view (FOV)**: defines the extent of a slice, e.g. 256mm x 256mm
 
 
+Specifics of MRI Data
+=====================
+
+MRI scanners output their neuroimaging data in a raw data format with which most analysis packages cannot work.  **DICOM** is a common, standardized, raw medical image format, but the format of your raw data may be something else; e.g., **PAR/REC** format from Philips scanners. Raw data is saved in `k-space <https://en.wikipedia.org/wiki/K-space_%28magnetic_resonance_imaging%29>`format, and it needs to be converted into a format that the analysis packages can use.  The most frequent format for newly generated data is called `NIfTI <http://nifti.nimh.nih.gov/>`_.  If you are working with older datasets, you may encounter data in **Analyze** format.  MRI data formats will have an **image** and a **header** part, and for NifTI format, they are in the same file, whereas in the older Analyze format, they are in separate files.
+
```

Comment:
I think it is relevant for people to know that NIfTI files have the ending ".nii" and Analyze always consist of two files that have the ending ".img" and ".hdr". Therefore, I would change your Text slightly to:

"MRI data formats will have an **image** and a **header** part, and for NifTI format, they are in the same file (``.nii``-file), whereas in the older Analyze format, they are in separate files (``.img`` and ``.hdr``-file)."
What do you think?